docs/source/en/model_doc/byt5.mdx

```diff
@@ -48,37 +48,93 @@ fine-tuning. If you are doing multi-task fine-tuning, you should use a prefix.
 ByT5 works on raw UTF-8 bytes, so it can be used without a tokenizer:
 
 ```python
-from transformers import T5ForConditionalGeneration
-import torch
+>>> from transformers import T5ForConditionalGeneration
+>>> import torch
 
-model = T5ForConditionalGeneration.from_pretrained("google/byt5-small")
+>>> model = T5ForConditionalGeneration.from_pretrained("google/byt5-small")
 
-input_ids = torch.tensor([list("Life is like a box of chocolates.".encode("utf-8"))]) + 3  # add 3 for special tokens
-labels = (
-    torch.tensor([list("La vie est comme une boîte de chocolat.".encode("utf-8"))]) + 3
-)  # add 3 for special tokens
+>>> input_ids = (
+...     torch.tensor([list("Life is like a box of chocolates.".encode("utf-8"))]) + 3
+>>> )  # add 3 for special tokens
+>>> labels = (
+...     torch.tensor([list("La vie est comme une boîte de chocolat.".encode("utf-8"))]) + 3
+>>> )  # add 3 for special tokens
```

Comment:
Can the comment go once above to avoid the formatting on several lines? And also maybe be more helpful because I have no idea what "add 3 for special tokens" means.